dcpy/library/templates/dcp_beaches.yml

```diff
@@ -0,0 +1,23 @@
+dataset:
+  name: dcp_beaches
+  acl: public-read
+  source:
+    url:
+      path: s3://edm-recipes/inbox/dcp_beaches/nyc_beaches.zip
```

Comment:
grim thought: I hope we never need to introduce versions for this dataset

Reply:
🙁